pkg/test/istio.io/examples/example.go

```diff
@@ -118,6 +118,7 @@ func (example *Example) Run() {
 			}
 			for _, step := range example.steps {
 				output, err := step.Run(kubeEnv, example.t)
+				example.t.Log(output)
```

Comment:
Duplicate with the same code below. Remove one of the `Log` calls. I would keep yours and remove the one below.

Reply:
Agree. And the call to example.t.Fatal(output) should be example.t.Fatal(err)

Reply:
This ```Log``` call is kept, and ```Fatal(output)``` is changed to ```Fatal(err)```. Thanks.